src/robot/parsing/model/statements.py

```diff
@@ -246,6 +324,23 @@ def value(self):
 class Metadata(DocumentationOrMetadata):
     type = Token.METADATA
 
+    @classmethod
+    def from_params(cls, metadata, separator=FOUR_SPACES, eol=EOL):
+        tokens = [
+            Token(Token.METADATA, 'Metadata'),
+            Token(Token.SEPARATOR, separator)
+        ]
+        metadata_lines = metadata.splitlines()
+        if metadata_lines:
+            tokens.append(Token(Token.ARGUMENT, metadata_lines[0]))
+            tokens.append(Token(Token.EOL, eol))
+        for line in metadata_lines[1:]:
+            tokens.append(Token(Token.CONTINUATION, '...'))
```

Comment:
Value isn't actually needed because `CONTINUATION` tokens get `'...'` automatically. Having the value visible perhaps make the code easier to understand, though, but then again e.g. `IF` code doesn't have it.

If this is changed, then possibly should also change `WITH_NAME` to use same approach. That type doesn't get the value automatically, but this could be easily changed in `Tken.__init__`. Feel free to do that as part of this PR if you think it makes sense.

Reply:
Right, and I actually used Token(Token.Continuation) in other parts of code. I will update WITH_NAME token as well